src/elements/content-preview/preview-header/PreviewHeaderComponent.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @flow
+ * @file Preview header container component
+ * @author Box
+ */
+
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames';
+import Header from './Header';
+import VersionsHeader from './VersionsHeader';
+import './Header.scss';
+
+type Props = {
+    canAnnotate: boolean,
+    canDownload: boolean,
+    contentOpenWithProps?: ContentOpenWithProps,
+    file?: BoxItem,
+    isPreviewingCurrentVersion: boolean,
+    onClose?: Function,
+    onDownload: Function,
+    onPrint: Function,
+    token: ?string,
+} & InjectIntlProvidedProps;
+
+const PreviewHeaderComponent = ({
+    canAnnotate,
+    canDownload,
+    contentOpenWithProps,
+    file,
+    isPreviewingCurrentVersion,
+    onClose,
+    onDownload,
+    onPrint,
+    token,
+}: Props) => {
+    const classes = classNames(['bcpr-header', !isPreviewingCurrentVersion && 'bcpr-header--dark']);
```

Comment:
This could be removed if we push the outer divs into the specific headers, but would cause repetition.